src/core/toga/widgets/table.py

```diff
@@ -101,8 +101,12 @@ def selection(self):
         """The current selection of the table.
 
         A value of None indicates no selection.
-        If the tree allows multiple selection, returns a list of
+        If the table allows multiple selection, returns a list of
         selected data nodes. Otherwise, returns a single data node.
+
+        The value of a column of the selection can be accessed with selection.header_name and selection.accessor_name
```

Comment:
This should always be `accessor_name`; the header name is human readable, and *may* match the accessor, but it isn't guaranteed to.

Reply:
OK, fixed